packages/core/src/external/carequality/ihe-gateway-v2/outbound/xcpd/process/schema.ts

```diff
@@ -51,34 +51,38 @@ export const PatientRegistryProfileSchema = z.object({
       .optional(),
   }),
   controlActProcess: z.object({
-    subject: z.object({
-      registrationEvent: z.object({
-        subject1: z.object({
-          patient: z.object({
-            id: z.object({
-              _root: z.string(),
-              _extension: z.string(),
-            }),
-            patientPerson: z.object({
-              addr: schemaOrArrayOrEmpty(AddressSchema).optional(),
-              name: schemaOrArray(NameSchema),
-              telecom: schemaOrArrayOrEmpty(TelecomSchema).optional(),
-              asOtherIDs: z.object({
-                id: schemaOrArrayOrEmpty(IdentifierSchema).optional(),
+    subject: z
+      .object({
+        registrationEvent: z.object({
+          subject1: z.object({
+            patient: z.object({
+              id: z.object({
+                _root: z.string(),
+                _extension: z.string(),
               }),
-              administrativeGenderCode: z
-                .object({
-                  _code: z.union([z.literal("F"), z.literal("M")]),
-                })
-                .optional(),
-              birthTime: z.object({
-                _value: z.string(),
+              patientPerson: z.object({
+                addr: schemaOrArrayOrEmpty(AddressSchema).optional(),
+                name: schemaOrArray(NameSchema),
+                telecom: schemaOrArrayOrEmpty(TelecomSchema).optional(),
+                asOtherIDs: schemaOrArrayOrEmpty(
+                  z.object({
+                    id: schemaOrArrayOrEmpty(IdentifierSchema).optional(),
+                  })
+                ).optional(),
+                administrativeGenderCode: z
+                  .object({
+                    _code: z.union([z.literal("F"), z.literal("M"), z.literal("UN")]),
```

Comment:
updating schema to accept UN. This came up in testing + looked at a  few historical responses in DB and we have unknown for gender. Also updated gender related types in a few other places to accomodate this change

Reply:
Yeah, we should def accept what the protocol/standard defines, and map it to our own.